monix-reactive/shared/src/main/scala/monix/reactive/internal/operators/IntersperseObservable.scala

```diff
@@ -0,0 +1,56 @@
+package monix.reactive.internal.operators
+
+import monix.execution.Ack.Continue
+import monix.execution.atomic.AtomicBoolean
+import monix.execution.{Ack, Cancelable}
+import monix.reactive.Observable
+import monix.reactive.observers.Subscriber
+
+import scala.concurrent.Future
+
+/**
+  * Created by omainegra on 5/29/17.
+  */
+private[reactive] final class IntersperseObservable[+A](source: Observable[A],
+                                                        separator: A) extends Observable[A]{ self =>
```

Comment:
I'm obsessed with indentation and I'd prefer for this to be consistent with the other files. Personally I don't like things dangling so far on the right, although I can understand the preference.